projects/admin/src/app/record/circulation-logs/circulation-log/circulation-log.component.ts

```diff
@@ -19,17 +19,28 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
 @Component({
   selector: 'admin-circulation-log',
   templateUrl: './circulation-log.component.html',
-  styleUrls: ['./circulation-log.component.css']
+  styleUrls: ['./circulation-log.component.scss']
 })
 export class CirculationLogComponent {
   /** Operation log record */
   @Input() record: any;
+  /** Is the log should be highligthed */
+  @Input() isHighlight = false;
+  /** Is the transaction must be separated from sibling elements */
+  @Input() separator = false;
 
   /** Event for close dialog */
   @Output() closeDialogEvent  = new EventEmitter();
 
   /** Circulation informations is collapsed */
   isCollapsed = true;
+  /** debugMode */
+  debugMode = false;
+
+  get canUseDebugMode(): boolean {
+    return true;
```

Comment:
Is it really usefull?

Reply:
You're right. I forget to correctly implement this getter ;-)